cb-event-forwarder.rpm.spec

```diff
@@ -1,69 +0,0 @@
-%define name cb-event-forwarder
```

Comment:
I wouldn't delete this file since RPM is their primary distro method right now. We may not use it but CB (and probably most of the people using it) still will.